browsers/qq_android.json

```diff
@@ -3,39 +3,39 @@
     "qq_android": {
       "releases": {
         "7.2.1": {
-          "relase_date": "2017-01-18",
+          "release_date": "2017-01-18",
           "status": "retired"
         },
         "7.3.1": {
-          "relase_date": "2017-03-08",
+          "release_date": "2017-03-08",
           "status": "retired"
         },
         "7.5": {
-          "relase_date": "2017-05-08",
+          "release_date": "2017-05-08",
           "status": "retired"
         },
         "7.5.1": {
-          "relase_date": "2017-05-18",
+          "release_date": "2017-05-18",
           "status": "retired"
         },
         "7.6.1": {
-          "relase_date": "2017-06-16",
+          "release_date": "2017-06-16",
           "status": "retired"
         },
         "7.8": {
-          "relase_date": "2017-08-21",
+          "release_date": "2017-08-21",
           "status": "retired"
         },
         "8.0": {
```

Comment:
Remove the trailing `.0` for consistency (see issue #1784).

Reply:
Done 👍